src/celutil/align.h

```diff
@@ -1,26 +1,28 @@
 #pragma once
 
-#include <type_traits>
+#include <cstddef>
+#include <cstdint>
+
+namespace celestia::util
+{
 
 /*! Returns aligned position for type T inside memory region.
  *  Designed for usage with functions which allocate unalligned
```

Comment:
please fix `unalligned` as you updated it in other location

Reply:
done